source/hud/roverSelector.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2014 Lockheed Martin Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may 
+// not use this file except in compliance with the License. You may obtain 
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, 
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+this.draw = function( context, position ) {
+    var rover, icon, posx, posy;
+    for ( var i = 0; i < this.rovers.length; i++ ) {
+        if ( this.rovers[ i ].enabled ) {
+            rover = this.rovers[ i ];
+            icon = this[ rover.id ];
```

Comment:
Not an issue - just a learning question. This is used for accessing an object in the node hierarchy by it's ID?